tslearn/docs/clustering.rst

```diff
@@ -0,0 +1,116 @@
+.. _clustering:
+
+Time Series Clustering
+======================
+
+Dedicated methods exist for time series clustering.
```

Comment:
I would say something like:

"Clustering is the task of grouping together similar objects. As neighbouring measurements in time series are correlated, dedicated methods are required. The Figure below illustrates this, by showing how the $k$-means algorithm does not produce meaningful results, due to the fact that it is not invariant to time shifts."